spec/factories/planning_application.rb

```diff
@@ -4,6 +4,8 @@
 
 FactoryBot.define do
   factory :planning_application do
+    before(:create) { |pa| pa.define_singleton_method(:audit_created) { true; } }
```

Comment:
what does this do?

Reply:
factory bot fails because it doesn't run the before create hooks by default, and so all planning applications were failing to create. The explanation / case for using the singleton method to allow for the planning application factory to create successfully is here:

https://stackoverflow.com/a/35562805

Reply:
I think it was actually failing not due to the before_create but due to there being nil users preventing the audit creation and thus the planning application creation, have added a bit more tolerance for nil values so the error went away - removed it